client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/standalone/StandaloneSonarLintEngine.java

```diff
@@ -19,13 +19,17 @@
  */
 package org.sonarsource.sonarlint.core.client.api.standalone;
 
+import java.util.Collection;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)